libs/types/hydration.ts

```diff
@@ -0,0 +1,11 @@
+interface HydrationData<Type = any> {
```

Comment:
If there's a single generic type parameter, I'd tend to call it `T`, because it's less likely to be confused with an actual type.